code/provision/playbooks/roles/ingress-controller/templates/values.yml.j2

```diff
@@ -0,0 +1,471 @@
+## nginx configuration
```

Comment:
I don't think this file needs to be the complete `vaules.yml` from the template. My understanding of helm is that a values file passed with `-f` can be a subset of this file where you are changing the defaults and doesn't have to redeclare everything. That way it is easy to see what you are changing while this way it is difficult to see the wood for the trees.
Apologies if I'm wrong about this, I haven't used helm that much.

Reply:
I had thought that too, I tried it but it didn't work so I left it, but having tried again as long as I also specify rbac as true (which should be true by default anyway...), it works fine - odd!

Looks like it was in a crashed state as the Helm chart hadn't created the cluster role binding for some reason, but deleted/re-ran it and seems to be working now.

Reply:
Nice, this is much clearer.